packages/table/preview/index.tsx

```diff
@@ -94,15 +100,34 @@ const ROW_COUNTS = [
     1000000,
 ];
 
+const CELL_CONTENTS = [
+    "Empty",
+    "CellNames",
+    "LongText",
+] as CellContent[];
+
 const COLUMN_COUNT_DEFAULT_INDEX = 2;
 const ROW_COUNT_DEFAULT_INDEX = 3;
 
+const ALPHANUMERIC_CHARS = "0123456789abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ";
+
+interface ICellContentGeneratorDictionary {
+    [key: string]: (rowIndex?: number, columnIndex?: number) => string;
```

Comment:
how about `Record<CellContent, (signature) => string>`? 

`Record` is TS's shorthand for this, a la `Pick` or `Partial`.